packages/endo/src/json.js

```diff
@@ -0,0 +1,7 @@
+export const parse = (source, location) => {
+  try {
+    return JSON.parse(source);
+  } catch (error) {
+    throw new SyntaxError(`Cannot parse JSON from ${location}, ${error}`);
```

Comment:
```suggestion
    if (error instanceof SyntaxError) {
      throw new SyntaxError(`Cannot parse JSON from ${location}, ${error}`);
    } else {
      throw error;
    }
```

See also my comments in #431 about whether the following approach might be better:

```suggestion
    if (someAppropriateOption) {
      console.log(`Cannot parse JSON from ${location},`, error);
    }
    throw error;
```